data-types.md

```diff
@@ -140,6 +143,9 @@ A standard point of vehicle telemetry. References to latitude and longitude impl
 | `battery_percent` | Integer         | Required if Applicable | Percent battery charge of vehicle, expressed between 0 and 100 |
 | `fuel_percent`    | Integer         | Required if Applicable | Percent fuel in vehicle, expressed between 0 and 100 |
 | `tipped_over`     | Boolean         | Required if Known      | If detectable and known, is the device tipped over or not? Default is 'false'. |
+| `stop_sequence` | Integer | Required if Applicable | Order of stops for a particular trip. The values must increase along the trip, but do not need to be consecutive. |
+| `route_id` | UUID | Required if Applicable | A unique route ID for the associated route and stops. Matches [GTFS](https://gtfs.org/documentation/schedule/reference/) `route_id` |
+| `service_id` | UUID | Required if Applicable | A unique service ID for the set of dates when service is available for a route. Matches [GTFS](https://gtfs.org/documentation/schedule/reference/) `service_id` |
```

Comment:
Just making a note here for the 'Files changed' view that these fields are for **Telemetry**.